conjur/init/conjurrc_data.py

```diff
@@ -16,6 +29,17 @@ def __init__(self, appliance_url = None, account = None, cert_file = None):
         self.cert_file = cert_file
         self.plugins = []
 
+    @classmethod
+    def load_conjurrc(cls):
```

Comment:
call this method `load_from_file`, and get the file path as parameter.
You can default it to `DEFAULT_CONFIG_FILE`, so you won't need to change the caller.

Reply:
👌 why do we want to get the path from outside?

Reply:
as in why is this better than a const?

Reply:
you can still use the const, but keep it as a parameter.
If one day you will want to change it to get the path from the user, you won't need to change much.
at the end, the class purpose is to read/write from the file, regardless of the path, so it shouldn't depend on it.